java/configuration/src/main/java/sleeper/configuration/properties/instance/EMRServerlessProperty.java

```diff
@@ -46,124 +47,144 @@ public interface EMRServerlessProperty {
             .propertyGroup(InstancePropertyGroup.BULK_IMPORT)
             .runCDKDeployWhenChanged(true).build();
     UserDefinedInstanceProperty BULK_IMPORT_EMR_SERVERLESS_EXECUTOR_CORES = Index
-            .propertyBuilder("sleeper.bulk.import.emr.serverless.executor.cores")
+            .propertyBuilder("sleeper.bulk.import.emr.serverless.spark.executor.cores")
             .description(
                     "The number of cores used by a Serverless executor. Used to set spark.executor.cores.\n"
                             + "See https://spark.apache.org/docs/latest/configuration.html.")
             .defaultValue("4")
             .propertyGroup(InstancePropertyGroup.BULK_IMPORT)
-            .runCDKDeployWhenChanged(true).build();
+            .runCDKDeployWhenChanged(false).build();
```

Comment:
I think it would be a bit clearer to leave the `runCDKDeployWhenChanged` value unset so it defaults to false, like we do with other properties.

Reply:
Removed